lib/benchee.ex

```diff
@@ -55,6 +55,14 @@ for {module, moduledoc} <- [{Benchee, elixir_doc}, {:benchee, erlang_doc}] do
 
     defp add_benchmarking_jobs(suite, jobs) do
```

Comment:
:wave: 

This function isn't the best one to add it to, this will only be triggered for `Benchee.run/2` - which is not the best as calling `Benchee.benchmark` manually is also a valid interface.

So this should rather live around `Benchee.benchmark` or some of the functions it calls: https://github.com/bencheeorg/benchee/blob/main/lib/benchee/benchmark.ex#L27-L36